### DISCUSSION TITLE
Where is the trash?

### DISCUSSION THREAD

**Discussion by EnochPrime** [immich-app/immich#1264] [Q&A]
From Android Immich app I selected a file (which was backed up) and selected delete. The app then asked to allow moving the file to a recycling bin which I allowed. Where would I find these "recycled" files?

**Comment by alextran1502** [comment]
It would be in your native gallery app trash

**Reply by titro** [reply]
this is not working for me. Deleted Objects are not visible on my Android Phone. Immich App and Server 1.41

Using a OnePlus Nord2 5G with Android 12

**Reply by alextran1502** [reply]
Then it depends on the platform then. For iOS it will go into the native Photo app's trash and be permanently deleted after 30 days
